tests/pr_test.go

```diff
@@ -2,43 +2,97 @@
 package test
 
 import (
+	"crypto/rand"
+	"encoding/base64"
+	"log"
+	"os"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/common"
 	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
 )
 
 // Use existing resource group
-const resourceGroup = "geretain-test-resources"
-const defaultExampleTerraformDir = "examples/default"
+const resourceGroup = "geretain-test-enterprisedb"
```

Comment:
Did you manually add this to our account? Please do not manually add permanent resource. They need to be added in https://github.ibm.com/GoldenEye/ge-dev-account-management/blob/master/main.tf and deployed with terraform

Reply:
You are right. I have not added yet. I have raised draft PR to create the resource group and EDB instances by this name. I guess I have to raise 2 PRs here. One for resource group name creation and another for provisioning EDB using main branch once changes are merged.

Reply:
I have added the default resource as `geretain-test-resources` and will be modifying this later once workspace issue is fixed.

Reply:
Sure. I will update this. Currently some issue is seen in workspace while provisioning new Resource group, hence using `geretain-test-resources` for now.